---
layout: index
title: Home
sidebar_sort_order: 1
---

<div class="panel">
<img src="{{site.baseurl}}/assets/img/profile_picture.png" alt="hi" class="inline" width="200"/>
<img src="{{site.baseurl}}/assets/img/cny.jpg" alt="Charlestown" class="inline" height="200" style="object-fit:cover"/>
</div>

<p>I am a PhD student being advised by <a href="https://main.pinellolab.partners.org/">Luca Pinello</a> at Harvard University <a href="https://dbmi.hms.harvard.edu/education/big-phd-program">Bioinformatics &amp; Integrative Genomics Program</a> and Massachusetts General Hospital. </p>
<br>

<div class="panel">
	<div class="panel-column">
		<img src="{{site.baseurl}}/assets/img/overview.jpg" alt="gene regulation" class="inline" style="height: 200px;"/>
		<p style="color:grey; font-size:12px">Uni and multi-modal single cell profile allows for studying gene regulatory relationship within and between modality. (Image: Li et al., 2020)</p>
	</div>
	<div class="panel-column">
		<p>My research interest lies in developing computational methods to better study the gene regulatory relationship. I work with <strong>single cell genomics</strong> and <strong>CRISPR screen data</strong> to study the relationship between genomic features and variant functions. </p>
<br>
	</div>
</div>

<div class="panel">
	<div class="panel-column">
		<img src="{{site.baseurl}}/assets/img/model_design.svg" alt="CRISPR editing" class="inline" style="height: 400px;object-fit:cover"/>
		<p style="color:grey;font-size:12px">CRISPR genome editing technology allows for direct perturbation of genetic variants. </figcaption>
	</div>
	<div class="panel-column">
		<p> CRISPR-Cas system is a powerful toolkit to directly test the variant effect to various phenotypes. I am actively developing a <a href="https://github.com/jykr/beige">Bayesian inference method</a> using <a href="http://pyro.ai/">probabilistic programming language</a> for CRISPR base-editor screen analysis that takes variable editing efficiency into account. <p>
		
		</p>
	</div>
</div>


<br>
<br>

<p>	Another axis of my research interest is to build <strong>interpretable statistical methods</strong> that harmonize multimodal and multi-level biological knowledge utilizing <strong>graph deep learning</strong> methods</p>

<div class="panel-column">
	<img src="{{site.baseurl}}/assets/img/emb.png" alt="SIMBA multimodal data embedding" class="inline" />
	<p style="color:grey">SIMBA, a graph representation learning based single cell embedding method allows joint analysis of cells, genes, motifs and peaks.</p>
</div>
