---
layout: page
title: Research
sidebar_link: true
sidebar_sort_order: 2
---

### Computational method development on CRISPR screen
*Aug 2021-*  
In collaboration with [Sherwood lab](https://sherwoodlab.bwh.harvard.edu/), I am analyzing and developing computational method to better identify the functional variants in CRISPR base editor screens.  

---

### SIMBA, a graph representation learning based single cell embedding method
*Jan 2021-*  
  

<img src="{{site.baseurl}}/assets/img/simba.png" alt="simba" class="inline" width="500" />
Our team developed [SIMBA](https://www.biorxiv.org/content/10.1101/2021.10.17.464750v1), a single cell embedding method that utilizes [PyTorch-BigGraph](https://github.com/facebookresearch/PyTorch-BigGraph), a multirelational graph embedding method in collaboration with Facebook Research.   

---


### Optimal transport based embedding of single cell genomics data
*May-Jul 2021*  

---

### Detecting structural variants from single cell whole genome amplification sequencing
*Aug-Nov 2021*  
Developed structural variant detection method from single cell whole genome sequencing data amplified with primary template-directed amplification (PTA) sampled from brain in [Park lab](https://compbio.hms.harvard.edu/index). 

---

### Prediction model from chromatin accessibility to gene expression
*Feb-Jul 2020*  
I analyzed fetal heart development SNARE-seq data to build and evaluate a shared prediction model from the chromatin accessibility to the gene expression in [Kharchenko lab](http://pklab.med.harvard.edu/).

---

### Single-cell expression profiling of metastatic bladder cancer
*Oct-Dec 2019*  
Identified heterogeneity in metastatic bladder cancer samples in [Van Allen lab](https://vanallenlab.dana-farber.org/).

---

### Characterizing the new class of super-enhancers that are shared across different cell types
*Jan-Oct 2019*  
  

<img src="{{site.baseurl}}/assets/img/constse.png" alt="constitutive super-enhancers" class="inline" width="400"/>
In [Jung lab](https://junglab.wixsite.com/home), I showed with our team that the constitutive super-enhancers are associated with the fast recovery of chromatin loops after cohesin disruption. Resulted in first author [publication](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2646-3) and oral presentation in [international conference](http://www.guolianglab.org/APBC2019/).

---

### Network analysis of a model for effective functional residue prediction from multiple sequence alignment
*Mar-Dec 2017*  
Advisor: Dongsup Kim
